nf_core/pipeline-template/.github/workflows/download_pipeline.yml

```diff
@@ -49,20 +49,20 @@ jobs:
         env:
           NXF_SINGULARITY_CACHEDIR: ./
         run: |
-          nf-core download ${{ env.REPO_LOWERCASE }} \
-          --revision ${{ env.REPO_BRANCH }} \
-          --outdir ./${{ env.REPOTITLE_LOWERCASE }} \
+          nf-core download {% raw %}${{ env.REPO_LOWERCASE }}{% endraw %} \
```

Comment:
How was this working before?

If there's no jinja template tags here, might be easier to have a single set of `{% raw %}` tags around the whole block / actions yaml.

Reply:
it was just previously added....